frontend/components/events/create/forms/BasicForm.tsx

```diff
@@ -226,8 +242,14 @@ export function BasicInformation({
                 value={capacityString}
                 type="number"
                 onChange={(e) => {
-                  setCapacityString(e.target.value);
-                  updateField({ capacity: parseInt(e.target.value) });
+                  const value = parseInt(e.target.value);
```

Comment:
Could we do some error handling for parseInt, so that our entire website does not break hehe.

Also, could you instead try use the Math.max function in typescript to achieve the below - do something like Math.max(capacity, 0). This just reduces some of the overhead code amount.

Reply:
i think i have fixed this